pages/services/prometheus/0.1.0-2.3.1/install/install.md

```diff
@@ -0,0 +1,299 @@
+---
+post_title: Install and Customize
+menu_order: 20
+enterprise: 'no'
+---
+
+ DCOS Prometheus is available in the Universe and can be installed by using either the web interface or the DC/OS CLI.
+
+The default DC/OS Prometheus Service installation provides reasonable defaults for trying out the service, but that may not be sufficient for production use. You may require different configurations depending on the context of the deployment.
+
+
+## Configuration Best Practices for Production
+  
+    - Install Alert Manager with base\first build of framework , its recommended to install remaining framework without alert manager and alert manager check box not to be checked, 
+    all the rest of the prometheus servers should point to the same alert manager which was installed with base\first build,to do this you would require to pass alert manager endpoint as target to your  prometheus servers.
+        
+    - Install global prometheus when required ,its recommended for global prometheus not to be configure until you require data to be federate from other salve prometheus servers,
+    To federate data from promethes slave server to global prometheus,slave prometheus server endpoints to be passed as target into global prometheus server under prometheus configuration yml.
+      
+## Prerequisites
+   
+- If you are using Enterprise DC/OS, you may [need to provision a service account](https://docs.mesosphere.com/1.10/security/ent/service-auth/custom-service-auth/) before installing DC/OS Prometheus Service. Only someone with `superuser` permission can create the service account.
+  - `strict` [security mode](https://docs.mesosphere.com/1.10/security/ent/service-auth/custom-service-auth/) requires a service account.
+  - In `permissive` security mode a service account is optional.
+  - `disabled` security mode does not require a service account.
+- Your cluster must have at least 3 private nodes.
```

Comment:
why is 3 private nodes a must? Also, doesnt seem to match whats configured in the framework.

Reply:
updated,building locally and will raise new PR , post making all the change